### ISSUE TITLE
Outline removed replaced with insufficient alternative?

### ISSUE THREAD

**Issue by Candu709** [Candu709/google-homepage#7]
As stated in https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

> [...]using outline: none without proper fallbacks makes your site significantly less accessible to any keyboard only user, not only those with reduced vision. Make sure to always give your interactive elements a visible indication of focus.

Outline removed
https://github.com/Candu709/google-homepage/blob/4bdff770eddf13dbe539605fb4e122bc5d57ba2a/styles/style.css#L96
Replacement 
https://github.com/Candu709/google-homepage/blob/4bdff770eddf13dbe539605fb4e122bc5d57ba2a/styles/style.css#L83-L85

Unsure if the shadow around counts as "proper fallback" without the addition of the search results dropdown.
A proposed solution would be to comment the line out for the time being.

**Comment by Candu709** [comment]
after this https://discord.com/channels/505093832157691914/690589233919819797/869987928447778878
I've decided its good enough as it is for now but it's definitively something to keep in mind for future projects #
